physionet-django/console/templates/console/event_agreement_detail.html

```diff
@@ -0,0 +1,18 @@
+{% extends "console/base_console.html" %} {% load static %} {% block title %}{{
+SITE_NAME }} - Event Agreements{% endblock %} {% block content %}
+<div class="card mb-3">
+  <div class="card-header">{{ event_agreement.name }} - edit</div>
+  <div class="card-body">
+    <form
+      method="post"
+      action="{% url 'event_agreement_detail' event_agreement.pk %}"
+    >
+      {% include "project/content_inline_form_snippet.html" with
+      form=event_agreement_form %}
```

Comment:
The template tag here `{% %}` here is spread across two lines - which doesn't work: https://code.djangoproject.com/ticket/8652

Reply:
Thank you.  i missed that. i was on my personal laptop and the linter messedup the tag.